nodemongobearer/controller/employees.js

```diff
@@ -0,0 +1,89 @@
+var express = require('express');
```

Comment:
@psinha20 Same mistakes - 

- Indentation
- Variable name
- Unnecessary package import
- Extra line
- Extra space
- Not give space after 'function()'
- Create the PR without proper code testing
- Copy-paste code from the previous assignments

cc @ketanTechracers